src/main/java/seedu/edrecord/logic/commands/DeleteGradeCommand.java

```diff
@@ -33,27 +33,25 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a grade from the student "
             + "identified by the index number used in the displayed student list. \n"
             + "Parameters: INDEX (must be a positive integer) "
-            + PREFIX_NAME + "ASSIGNMENT NAME \n"
-            + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_NAME + "Assignment 2";
+            + PREFIX_ID + "ASSIGNMENT ID \n"
+            + "Example: " + COMMAND_WORD + " 1 " + PREFIX_ID + "4";
 
     public static final String MESSAGE_SUCCESS = "Deleted grade %s \nfor assignment: %s \nfor student: %s";
-    public static final String MESSAGE_NO_SUCH_ASSIGNMENT = "There is no assignment with this name";
-    public static final String MESSAGE_NO_SUCH_GRADE = "There is no grade for this assignment";
+    public static final String MESSAGE_NO_SUCH_GRADE = "There is no grade for this assignment.";
 
     private final Index index;
-    private final Name name;
+    private final Index id;
```

Comment:
Might be clearer to rename as `studentIndex` and `assignmentId` but not a major issue.

Reply:
Fixed.